graph_model/__init__.py

```diff
@@ -0,0 +1 @@
+from .model import GraphModel
```

Comment:
```suggestion
from .model import GraphModel as Model
```
Will make import interchangeable with other models.